### PR TITLE
[New Rule] Persistence via Hidden Run Key ValName

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/533

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

**Comment by Samirbous** [comment]
**Alert Example:**

image

File: rules/windows/persistence_via_hidden_run_key_valuename.toml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/15"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies a persistence trick to confuse DFIR investigation that uses NtSetValueKey native API to create a hidden (null
+terminated) registry key.An Adversary may use this method to hide his persistence entry from system utilities such as
+Regedit.
```
Comment (bm11100) [inline rules/windows/persistence_via_hidden_run_key_valuename.toml:12]:
```suggestion
Identifies a persistence mechanism that utilizes the NtSetValueKey native API to create a hidden (null
terminated) registry key. An adversary may use this method to hide from system utilities such as
the Registry Editor (regedit).
```

Comment (bm11100) [reply rules/windows/persistence_via_hidden_run_key_valuename.toml:12]: Probably need to re-lint afterwards

**Review by bm11100 — APPROVED** [review]
‼️

**Review by threat-punter — APPROVED** [review]
👍

**Pull request merged**